app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+<div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- hey, please, let's make our design look more like the wireframe to meet the project requirement below:

*your design*
![image](https://github.com/0sugo/blog-app/assets/59397159/43e87646-af3e-4d96-89e7-4294f94d4060)

*the wireframe*
![image](https://github.com/0sugo/blog-app/assets/59397159/43123442-6039-45ff-b219-0d52673d1ad9)

Reply:
Hi @zieeco  Nice job reviewing my PR However did you note the difference between a sign up page  and login page?

Reply:
- Well resolved 
<img width="1050" alt="Screenshot 2023-07-27 at 11 34 53" src="https://github.com/0sugo/blog-app/assets/58692590/e84a8f4d-2f56-4aa3-841b-1c241549d87f">
<img width="302" alt="Screenshot 2023-07-27 at 11 49 13" src="https://github.com/0sugo/blog-app/assets/58692590/c36f9a69-c7ee-4b43-adcf-eb7849acc8c6">